src/ext.js

```diff
@@ -238,6 +238,59 @@ export default function ext(/* env */) {
                   dualOutput: true,
                   show: true,
                 },
+                {
+                  component: 'item-selection-list',
+                  type: 'string',
+                  ref: 'style.textStyle',
+                  translation: 'Text Style',
+                  horizontal: true,
+                  multipleSelect: true,
+                  items: [
+                    {
+                      component: 'icon-item',
+                      icon: 'bold',
+                      value: 'bold',
+                    },
+                    {
+                      component: 'icon-item',
+                      icon: 'italic',
+                      value: 'italic',
+                    },
+                    {
+                      component: 'icon-item',
+                      icon: 'underline',
+                      value: 'underline',
+                    },
+                  ],
+                  defaultValue: {
+                    bold: true,
+                  },
+                },
+                {
+                  component: 'item-selection-list',
+                  type: 'string',
+                  ref: 'style.textAlign',
+                  translation: 'properties.Alignment',
+                  horizontal: true,
+                  items: [
+                    {
+                      component: 'icon-item',
+                      icon: 'align_left',
+                      value: 'left',
+                    },
+                    {
+                      component: 'icon-item',
+                      icon: 'align_center',
+                      value: 'center',
+                    },
+                    {
+                      component: 'icon-item',
+                      icon: 'align_right',
+                      value: 'right',
+                    },
+                  ],
+                  defaultValue: 'center',
```

Comment:
The default value is both here and in style-formatter, maybe one should be removed?

Reply:
Good catch. Yes, we need to keep it here though, since we need to show the correct value in the pp. Actually that should go for background defaults as well. Just noticed that the position does not have a value until you choose one.

Reply:
Should all be stored in object.properties. I made a new user story for it to move all initials values there